CommonLib/Error/GREYErrorFormatter.m

```diff
@@ -110,13 +110,11 @@ BOOL GREYShouldUseErrorFormatterForExceptionReason(NSString *reason) {
   
   NSString *assertionCriteria = error.userInfo[kErrorDetailAssertCriteriaKey];
   if (assertionCriteria) {
-    [logger addObject:[NSString stringWithFormat:@"%@: %@", kErrorDetailAssertCriteriaKey,
```

Comment:
On checking this, we could just change kErrorDetailAssertCriteriaKey to "Failed Assertion" and kErrorDetailActionNameKey to "Failed Action". I can send a cl for this separately and you could merge it into your PR.

Reply:
Good idea. Updated the PR, and adjusted the 3 tests that would've failed from the change.